static/js/components/inputs/SelectField.js

```diff
@@ -12,8 +13,11 @@ import type {
   UpdateProfileFunc,
 } from '../../flow/profileTypes';
 
+const CREATE_OPTION_REGEX = /^Create option \"|\"$/g;
```

Comment:
I'm fine using `replace` as long as we have tests for various inputs that match against the regex. By the way my IDE says that the `\` in the `\"` in the regex are unnecessary